src/SecurityChecker/Command/SecurityCheckerCommand.php

```diff
@@ -1,12 +1,134 @@
 <?php
-declare(strict_types=1);
+declare(strict_types = 1);
+
 /**
  * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
-namespace Brancho\SecurityChecker;
 
-class SecurityCheckerCommand
+namespace SecurityChecker\Command;
+
+use SecurityChecker\Result;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+class SecurityCheckerCommand extends Command
 {
+    protected const COMMAND_NAME = 'security:check';
+
+    protected const BINARY_CHECKER = 'https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_amd64';
```

Comment:
Even though it's not a Spryker module, I'd expect these kinds of things to be passed as config or something like that. Does it make sense?

Reply:
For simplicity this is probably fine for now.